src/custom/state/fee/hooks.ts

```diff
@@ -1,19 +1,10 @@
-import { ChainId } from '@uniswap/sdk'
 import { useCallback } from 'react'
 import { useDispatch, useSelector } from 'react-redux'
 
 import { AppDispatch, AppState } from 'state'
-import { updateFee, clearFee } from './actions'
+import { updateFee, clearFee, AddFeeParams, ClearFeeParams } from './actions'
 import { FeeInformation, FeesMap } from './reducer'
 
-interface AddFeeParams extends ClearFeeParams {
-  fee: FeeInformation
-}
-interface ClearFeeParams {
-  token: string // token address,
-  chainId: ChainId
-}
-
 type AddFeeCallback = (addTokenParams: AddFeeParams) => void
 type ClearFeeCallback = (clearTokenParams: ClearFeeParams) => void
```

Comment:
Since you are here, update the parameters to be inline with the type:

```suggestion
type AddFeeCallback = (addFeeParams: AddFeeParams) => void
type ClearFeeCallback = (clearFeeParams: ClearFeeParams) => void
```